tests/uploader_test.py

```diff
@@ -289,13 +289,14 @@ def test_responsive_breakpoints(self):
             responsive_breakpoints=[dict(create_derived=False,
                                          transformation=dict(angle=90)),
                                     dict(create_derived=False,
-                                         transformation=dict(angle=45))])
+                                         transformation=dict(angle=45),
+                                         format="jpg")])
```

Comment:
Add test for empty string in `format`

Reply:
Please review https://github.com/cloudinary/pycloudinary/pull/125/commits/e8aec4fb5d977be45cc79dd147399ce0bc9ac478